src/components/v2/values/ea/EAValues.component.tsx

```diff
@@ -282,24 +336,14 @@ function ExternalAppValues({ appId }: { appId: string }) {
                                 installedAppInfo={installedAppInfo}
                                 releaseInfo={releaseInfo}
                             />
-                            {installedAppInfo && <StaticChartRepoInput releaseInfo={releaseInfo} />}
```

Comment:
is it intentional?

Reply:
Yes. We're not going to show the static field anymore.